src/libraries/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -537,4 +537,7 @@
   <data name="net_http_qpack_no_dynamic_table" xml:space="preserve">
     <value>The HTTP/3 server attempted to reference a dynamic table index that does not exist.</value>
   </data>
-</root>
+  <data name="net_http_request_timedout" xml:space="preserve">
+    <value>The request timed out.</value>
```

Comment:
I think it might be helpful to include the timeout information in the message, e.g.
```
The request was canceled due to the configured HttpClient.Timeout of {0} seconds elapsing.
````
or something like that.  In the past, some folks I've spoken with haven't been aware of the timeout, of its default value, etc.

Reply:
Done